src/form/birth/certificate-handlebars.ts

```diff
@@ -29,12 +29,8 @@ export const certificateHandlebars = {
   birthConfigurableIdentifier1: 'birthConfigurableIdentifier1',
   birthConfigurableIdentifier2: 'birthConfigurableIdentifier2',
   birthConfigurableIdentifier3: 'birthConfigurableIdentifier3',
-  placeOfBirth: 'placeOfBirth', // equal to either the name of a HEALTH_FACILITY or undefined
   placeOfBirthFacility: 'placeOfBirthFacility', // equal to either the name of a HEALTH_FACILITY or undefined
-  placeOfBirthCountry: 'placeOfBirthCountry', // used where event occurs in a HEALTH_FACILITY
-  placeOfBirthDistrict: 'placeOfBirthDistrict', // used where event occurs in a HEALTH_FACILITY
-  placeOfBirthState: 'placeOfBirthState', // used where event occurs in a HEALTH_FACILITY
-  countryPlaceofbirth: 'countryPlaceofbirth', // THE FOLLOWING are used where event occurs in a PRIVATE_HOME or OTHER therefore placeOfBirth is undefined
```

Comment:
@euanmillar We can use the `statePlaceofbirth`, `districtPlaceofbirth` etc even with the health facility so I'm removing them from here. We can remove them core when we work on `healthFacilityId`

Reply:
https://github.com/opencrvs/opencrvs-core/issues/6527